src/backends/sentry_backend_crashpad.cpp

```diff
@@ -324,16 +324,17 @@ sentry__crashpad_backend_shutdown(sentry_backend_t *backend)
 }
 
 static void
-sentry__crashpad_backend_add_breadcrumb(
-    sentry_backend_t *backend, sentry_value_t breadcrumb)
+sentry__crashpad_backend_add_breadcrumb(sentry_backend_t *backend,
+    sentry_value_t breadcrumb, const sentry_options_t *options)
 {
     crashpad_state_t *data = (crashpad_state_t *)backend->data;
 
-    bool first_breadcrumb = data->num_breadcrumbs % SENTRY_BREADCRUMBS_MAX == 0;
+    size_t max_breadcrumbs = options->max_breadcrumbs;
+
+    bool first_breadcrumb = data->num_breadcrumbs % max_breadcrumbs == 0;
```

Comment:
To confirm: Can `max_breadcrumbs ever be `0`?

Reply:
good point. Code in other places was failing with `max_breadcrumbs = 0`, I fixed that up and its now correctly supported.